cmd/configure.go

```diff
@@ -499,6 +503,39 @@ func configureGithub(ctx context.Context, _flags ConfigureGithubFlags) error {
 
 	ctx = events.SetTargetInContext(ctx, workingDir)
 
+	// check if the git repository is a github URI
+	event := shared.CliEvent{}
+	events.EnrichEventWithGitMetadata(ctx, &event)
```

Comment:
nit: I feel like this is a bit of a hack. Can we just expose a method like `GetGitMetadata` that is used both inside `EnrichEvent` and here?

Reply:
Ya I was just trying to not change it in core, I can do that